spock-core/src/main/java/org/spockframework/compiler/AstUtil.java

```diff
@@ -329,15 +337,7 @@ public static Expression getVariableType(BinaryExpression assignment) {
     return type == null || type == ClassHelper.DYNAMIC_TYPE ? ConstantExpression.NULL : new ClassExpression(type);
   }
 
-  public static MethodCallExpression createGetAtMethod(Expression expression, int index) {
-    return new MethodCallExpression(expression,
-                                    GET_AT_METHOD_NAME,
-                                    new ConstantExpression(index));
-  }
-
-  public static MethodCallExpression createGetMethod(Expression expression, int index) {
-    return new MethodCallExpression(expression,
-                                    GET_METHOD_NAME,
-                                    new ConstantExpression(index));
+  public static MethodCallExpression createGetAtMethodCall(Expression expression, int index) {
```

Comment:
Why don't we nee the normal `get` anymore?

Reply:
Because this is now done as direct call: https://github.com/spockframework/spock/pull/1201/files#diff-93ae30b9276f36a9d9554475ec30262eL529-R537